### PR TITLE
begin korean documentation work

### PR DESCRIPTION
**Pull request by jeayoon**

### CODE REVIEW

**Comment by jeayoon** [comment]
Hi @ravage84 ,
so far we made some progress on the documentation and we sent a pull request over to you. 
Could you check if that went through and confirm it to allow us to continue?
 If there are any issues, we would like some feedback! 
Thank You

**Comment by ravage84** [comment]
Hi, @jeayoon 

Thank you for your work.

Just to make it clear: I'm not the only "gatekeeper" of the documentation. This is a community & team effort.

The CakePHP documentation/core team is currently discussing how to progress and we will get back at you.

File: kr/intro/where-to-get-help.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+테스트 케이스
+============
+
+API에서 제공되는 정보가 충분하지 않다고 느끼는 경우 CakePHP에서 제공하는 테스트 케이스 코드를 확인할 수 있습니다.
+함수의 실용적인 예제와 클래스의 멤버를 사용하는 방법으로 사용할 수 있습니다.::
+
+tests/TestCase/
+
+IRC チャンネル
```
Comment (jeayoon) [inline kr/intro/where-to-get-help.rst]: It should be corrected

File: kr/tutorials-and-examples/cms/authentication.rst [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+            if (strlen($value)) {
+                $hasher = new DefaultPasswordHasher();
+
+                return $hasher->hash($value);
+            }
+        }
+    }
+
+여기서 브라우저에서  **http://localhost:8765/users** 에 액세스하여 사용자의 목록 봅시다.
+ :doc:`インストール <installation>`에 만들어진 기본 사용자를 편집 할 수 있습니다.
```
Comment (jeayoon) [inline kr/tutorials-and-examples/cms/authentication.rst]: It should be corrected

File: kr/intro/cakephp-folder-structure.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+Controller
+    어플리케이션의 컨트롤러와 컴포넌트를 포함합니다.
+Locale
+    국제화를 위한 문자열 파일을 저장합니다.
+Model
+    어플리케이션의 테이블, 엔티티, 동작을 포함합니다.
+Shell
+    어플리케이션에서 사용할 콘솔 명령이나 콘솔 작업이 포함됩니다.
+    상세 설명은 :doc:`/console-and-shells` 를 확인하시길 바랍니다.
```
Comment (jeayoon) [inline kr/intro/cakephp-folder-structure.rst]: It should be corrected

File: kr/installation.rst [hunk 4]
```diff
@@ -0,0 +1,480 @@
...
+--------------------------
+
+이상으로 Composer 다운로드와 설치가 완료되었습니다. my_app_name 디렉토리에
+CakePHP의 새로운 어플리케이션을 생성해주세요. 아래의 composer 명령어를 실행하여 생성합니다.
+
+.. code-block:: bash
+
+    php composer.phar create-project --prefer-dist cakephp/app my_app_name
+
+または Composer にパスが通っているのであれば下記のコマンドも使えます。
```
Comment (chinpei215) [inline kr/installation.rst]: This should be translated into Korean.

File: kr/installation.rst [hunk 5]
```diff
@@ -0,0 +1,480 @@
...
+	                      appendQueryString="true" />
+	                </rule>
+	            </rules>
+	        </rewrite>
+	    </system.webServer>
+	</configuration>
+
+일단 IIS에서 사용할 수있는 재 작성 규칙을 포함하는 web.config 파일이 되었으면 CakePHP 링크, CSS, JavaScript, 재 라우팅 (rerouting)은 제대로 작동하는 것입니다.
+
+URL リライティングを使わない場合
```
Comment (chinpei215) [inline kr/installation.rst]: This should be translated into Korean.

File: kr/tutorials-and-examples/bookmarks/intro.rst [hunk 6]
```diff
@@ -0,0 +1,407 @@
...
+
+.. note::
+
+    CakePHP의 기본 설정 파일은**config/app.default.php**에 있습니다.
+
+Scaffold 코드 작성
+=====================
+
+데이터베이스가 CakePHP의 명명 규칙에 따르고 있으므로 기본적인 응용 프로그램을
+빠르게 생성하기 위해  :doc:`bake コンソール </bake/usage>`응용 프로그램을 사용할 수 있습니다.
```
Comment (chinpei215) [inline kr/tutorials-and-examples/bookmarks/intro.rst]: This should be translated into Korean.

File: kr/tutorials-and-examples/bookmarks/intro.rst [hunk 7]
```diff
@@ -0,0 +1,407 @@
...
+        } else {
+            $bookmarks
+                ->innerJoinWith('Tags')
+                ->where(['Tags.title IN ' => $options['tags']]);
+        }
+
+        return $bookmarks->group(['Bookmarks.id']);
+    }
+
+:ref:`カスタム Finder メソッド <custom-find-methods>` 을 구현했습니다.
```
Comment (chinpei215) [inline kr/tutorials-and-examples/bookmarks/intro.rst]: This should be translated into Korean.

File: kr/tutorials-and-examples/bookmarks/part-two.rst [hunk 8]
```diff
@@ -0,0 +1,395 @@
+북마크 튜토리얼 파트 2
+##################################
+
+:doc:`チュートリアルの前編 </tutorials-and-examples/bookmarks/intro>` 을 끝내면,
```
Comment (chinpei215) [inline kr/tutorials-and-examples/bookmarks/part-two.rst]: This should be translated into Korean.

File: kr/tutorials-and-examples/cms/articles-controller.rst [hunk 9]
```diff
@@ -0,0 +1,489 @@
...
+    그래서 컨트롤러에서 ``allowMethod()`` 를 사용했습니다.
+
+단지 로직을 실행하고 다른 액션으로 리디렉션하기 때문에 이 액션에는 템플릿이 없습니다.
+사용자가 Article을 삭제할 수있는 링크로 색인 템플릿을 업데이트 할 수 있습니다.
+
+.. code-block:: php
+
+    <!-- File: src/Template/Articles/index.ctp  (삭제링크 추가) -->
+
+    <h1>記事一覧</h1>
```
Comment (chinpei215) [inline kr/tutorials-and-examples/cms/articles-controller.rst]: This should be translated into Korean.

File: kr/tutorials-and-examples/cms/articles-controller.rst [hunk 10]
```diff
@@ -0,0 +1,489 @@
...
+            </td>
+            <td>
+                <?= $article->created->format(DATE_RFC850) ?>
+            </td>
+            <td>
+                <?= $this->Html->link('편집', ['action' => 'edit', $article->slug]) ?>
+                <?= $this->Form->postLink(
+                    '삭제',
+                    ['action' => 'delete', $article->slug],
+                    ['confirm' => 'よろしいですか?'])
```
Comment (chinpei215) [inline kr/tutorials-and-examples/cms/articles-controller.rst]: This should be translated into Korean.

File: kr/tutorials-and-examples/cms/database.rst [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+데이터베이스 구성
+===================
+
+다음으로, CakePHP에서 데이터베이스가 어디에 있는지, 어떻게 연결하는지 설명 해 보겠습니다.
+**config/app.php** 파일의 ``Datasources.default`` 배열에있는 값을 설정에 적용되는 값으로 바꿉니다.
+완료된 구성 배열의 샘플은 다음과 같습니다. ::
+
+    <?php
+    return [
+        // 上には他の設定があります
```
Comment (chinpei215) [inline kr/tutorials-and-examples/cms/database.rst]: This should be translated into Korean.

**Comment by markstory** [comment]
@jeayoon Is this ready to be merged?

**Comment by jeayoon** [comment]
@markstory Can I check it out one more time? and if confirmed I will let you know next week.

**Comment by jeayoon** [comment]
@markstory I have confirmed Korean documentation. and I ready to merge this branch!

**Pull request merged**